style.css

```diff
@@ -448,45 +368,36 @@ li {
   justify-content: space-around;
   flex-direction: row;
   width: 140.17px;
-  height: 24px;
   order: 4;
 }
 
 .connect_text2 {
-  gap: 8px;
-  width: 117px;
-  height: 24px;
+  gap: 1vh;
 }
 
 .connect_text2 h2 {
   font-family: 'Poppins', sans-serif !important;
   font-style: normal;
   font-weight: 500;
-  font-size: 16px;
-  line-height: 14px;
+  font-size: 4vw;
   color: #7f8cff;
-  flex: none;
 }
 
 .resume_link {
   align-items: center;
-  padding: 12px;
-  gap: 10px;
-  width: 162px;
-  height: 48px;
+  padding: 2.5vw;
   border: 1px solid #6070ff;
```

Comment:
- [ ] Please, add interaction effects to this one too! 🤖 Remember that these little details make the difference when a user is navigating through your page, they improve a lot their experience! Consider adding them. 🤓 🛠️ 

![image](https://user-images.githubusercontent.com/77069011/200000030-d4a752f2-f751-48de-b5f1-5a8529526b81.png)